client/src/components/CredentialsButton/CredentialsButton.css

```diff
@@ -0,0 +1,18 @@
+.credentials {
```

Comment:
It would be better to use CSS modules where file name is "CredentialsButton.module.css" instead.

e.g. import styles from "./CredentialsButton.css"
<div className={styles.credentials}>
    {other_content}
</div>

By using CSS modules, it allows us to reduce errors instead of just typing in CSS classNames as strings

Reply:
Fixed (Nestor): changed to .module